src/lib/frontend/d_cnf.ml

```diff
@@ -444,11 +444,38 @@ let smt_fpa_builtins =
       end
     | _ -> `Not_found
 
+let lemma_attr : string DStd.Tag.t = DStd.Tag.create ()
+
+type _ Dl.Typer.T.err +=
+  | Expect_simple_sexpr
```

Comment:
Nit: This is not very explicit about where/why we expect a simple sexpr — `Invalid_lemma_name` or `Non_simple_symbol_lemma_name` would be more explicit.

Reply:
Naming is so hard. I wasn't a big fan of this one too.